pyproject.toml

```
@@ -19,9 +19,9 @@ include = ["sansio_lsp_client/py.typed"]
 [tool.poetry.dependencies]
 python = ">=3.10"
 dataclasses = { version = "^0.8", python = ">=3.6,<3.7" }
-pydantic = "^1.10.14"
 black = "^24.4.2"
 mypy = "^1.10.0"
+pydantic = ">=2.0.0"
```

If pydantic 3 is released, we don't want to start using it automatically and break everything:

```suggestion
pydantic = ">=2.0.0,<3.0.0"
```

---

🤦 .... 
Done 😄